src/service_control_client_impl_quota_test.cc

```diff
@@ -109,14 +122,15 @@ class ServiceControlClientImplQuotaTest : public ::testing::Test {
   MockQuotaTransport mock_quota_transport_;
   MockReportTransport mock_report_transport_;
 
-  std::unique_ptr<ServiceControlClient> client_;
+  std::unique_ptr<ServiceControlClient> client_cached_;
+  std::unique_ptr<ServiceControlClient> client_non_cached_;
 };
 
 // Error on different service name
 TEST_F(ServiceControlClientImplQuotaTest, TestQuotaWithInvalidServiceName) {
-  ServiceControlClientOptions options(CheckAggregationOptions(1, 500, 1000),
-                                      QuotaAggregationOptions(1, 500),
-                                      ReportAggregationOptions(1, 500));
+  ServiceControlClientOptions options(CheckAggregationOptions(10, 500, 1000),
+                                      QuotaAggregationOptions(10, 500),
+                                      ReportAggregationOptions(10, 500));
```

Comment:
This is required. The client instance for this case requires different service name for initialization.